ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/monitor-layer1.j2

```diff
@@ -0,0 +1,23 @@
+{#
+ Copyright (c) 2023 Arista Networks, Inc.
+ Use of this source code is governed by the Apache License 2.0
+ that can be found in the LICENSE file.
+#}
+{# eos - monitor layer1 #}
+{% if monitor_layer1.logging_mac_fault is arista.avd.defined(true) or monitor_layer1.logging_transceiver is arista.avd.defined %}
+!
+monitor layer1
+{%     if monitor_layer1.logging_transceiver.enable is arista.avd.defined(true) %}
+{%         if monitor_layer1.logging_transceiver.dom is arista.avd.defined(true) %}
+   logging transceiver dom
+{%         endif %}
+{%         if monitor_layer1.logging_transceiver.communication is arista.avd.defined(true) %}
+   logging transceiver communication
+{%         elif monitor_layer1.logging_transceiver.dom is not arista.avd.defined %}
+   logging transceiver
+{%         endif %}
```

Comment:
This looks weird 

```
host3(config-mon-layer1)#show active
monitor layer1
   logging transceiver communication
   logging mac fault
```

Can you please share a device where this behavior is actually respected? (I am trying on cEOS)